packages/shared/src/utils/user-display-name.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Get user display name from multiple fields
+ */
+export const getUserDisplayName = ({
```

Comment:
Can you also update the usage in `console` to make sure both console and core have the same reference?